seismic-app/src/components/Map/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import GoogleMapReact from 'google-map-react';
```

Comment:
While Google's map is full of features, we're probably going to avoid it since it incurs a big cost ([especially for an NGO](facebook.com/VotDiaspora/photos/a.310080362671361/998482720497785))
We're planning on using [Here maps](https://developer.here.com/), so we're probably going to go for [this React package](https://www.npmjs.com/package/here-maps-react)